test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -362,6 +362,35 @@
          (finally
            (jruby-testutils/return-instance jruby-service jruby-instance :settings-plumbed-test)))))))
 
+(deftest jruby-environment-vars-test
+  (testing "Make sure that the environment variables whitelisted in puppetserver.conf are being set"
+    (tk-testutils/with-app-with-config
+     app
+     [jruby/jruby-puppet-pooled-service
+      profiler/puppet-profiler-service
+      jruby-utils/jruby-pool-manager-service]
+     (jruby-testutils/jruby-puppet-tk-config
+      (jruby-testutils/jruby-puppet-config
+       {:ruby-load-path  jruby-testutils/ruby-load-path
+        :gem-home        jruby-testutils/gem-home
+        :master-conf-dir jruby-testutils/conf-dir
+        :master-code-dir jruby-testutils/code-dir
+        :master-var-dir  jruby-testutils/var-dir
+        :master-run-dir  jruby-testutils/run-dir
+        :master-log-dir  jruby-testutils/log-dir
+        :environment-vars jruby-testutils/environment-vars}))
```

Comment:
Not totally sure but it looks like all these config options are being set to the values `jruby-puppet-config` already spits out. Might not need them here